tensorflow/core/kernels/mkl_fused_ops_test.cc

```diff
@@ -168,6 +168,55 @@ class MklFusedConv2DOpTest : public OpsTestBase {
     CommonTestUtilities<T>::RunAndFetch(root, "with_relu", output);
   }
 
+  void RunConv2DWithBiasAndAdd(const Tensor& input_data,
```

Comment:
This is getting verbose. Can we merge `RunConv2DWithBias`, `RunConv2DWithBiasAndRelu`, `RunConv2DWithBiasAndAdd` and `RunConv2DWithBiasAndAddRelu` into just one function (i.e., `RunConv2DUnfused`) that takes an op list?